README.md

```diff
@@ -1,2 +1,2 @@
 # test
-Test repository.  Not for testing.
+Test repository.  For testing.
```

Comment:
```suggestion
Test repository.  For not testing.
```

Reply:
I don't appear to be able to commit this suggestion, so it may be that I can only merge it as is rather than being able to update.  Will try from a real checkout.

Reply:
I don't appear to have any way to push to this branch, though `gh pr checkout` was able to check it out.